procedures/MT/slew/setup.ipynb

```diff
@@ -0,0 +1,988 @@
+{
```

Comment:
<p>Doesn't this command fail and we need to run twice? I know it is not the final state, but it might be good to comment to run it twice or have it in a try/except for now. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>